data/css/modules/card/_list.scss

```diff
@@ -87,4 +87,16 @@
     .composite &.width-e &__synopsis {
         font-size: 18.8px;  /* 16px spec * 88% because spec was at 110% scale not 125% */
     }
+
+    // Video/audio icon sizing
+    &.width-f,
```

Comment:
I don't see this in the specs either, but even so shouldn't it be the small size all the way down to `.width-a`?